### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by rhagarty**
- changed some organization
- added pull of dataset into dsx
- removed some unused images

### CODE REVIEW

**Review by stevemar — APPROVED** [review]
i like these changes, before merging i'd like @vinodhmohan91 to take a look.

**Comment by rhagarty** [comment]
- updated architecture diagram and accompanying flow text
- moved original flow text up to section where we detail each of the notebooks

**Comment by rhagarty** [comment]
fixes #7

File: README.md [hunk 1]
```diff
@@ -24,63 +26,78 @@ In order for the DSX Local to be able to utilize the resources (spark, compute)
...
 
-This repo contains 8 jupyter notebooks and 6 scripts that are used to illustarte the three scenarios mentioned above.
+This code pattern contains 8 jupyter notebooks and 6 scripts. Here is a view of the notebooks as shown by the DSX Local UI:
 
 ![](doc/source/images/Jupyter-notebooks-list.png)
 
-**Scenario 1 (Run pyspark model in remote HDP cluster)**: <br>
-- Spam Filter on local spark.ipynb (pyspark in local) <br>
-- Spam Filter on remote spark.ipynb (pyspark in HDP) <br>
+As mentioned earlier, this code pattern offers three examples of how to develope, train, and deploy a Spam Filter Model. For each example, multiple notebooks will be provided to show both a local (DSX Local) and remote (HDP cluster) solution. The following lists the associated notebooks for each:
```
Comment (markstur) [inline README.md]: s/develope/develop/

File: README.md [hunk 2]
```diff
@@ -24,63 +26,78 @@ In order for the DSX Local to be able to utilize the resources (spark, compute)
...
+
+* Develop and train a Spam Filter using pyspark.
+
+  `"Spam Filter on local spark"`
+    - Load the spam collection dataset using spark context in DSX Local.
+    - Use Spark Data Pipeline to extract the TF-IDF features and use Spark MLlib to train the Spam Filter pyspark model locally.
```
Comment (markstur) [inline README.md:41]: nit - I see a mix of "Spark" and "spark".  Some look intentional but there are some that I can't explain so you might want to take a pass at those.

**Review by markstur — CHANGES_REQUESTED** [review]
just one typo and another comment you might want to look at

File: README.md (hunk 1 above)
Comment (rhagarty) [reply README.md]: will fix

File: README.md (hunk 2 above)
Comment (rhagarty) [reply README.md:41]: I'm going with uppercase

**Review by markstur — APPROVED** [review]

**Comment by rhagarty** [comment]
Changed all DSX Local references to Watson Studio Local.
Still TODO:
- update screen shots
- determine if DSXHI is still needed or has been replaced by another service
- update architecture diagram

**Comment by vinodhmohan91** [comment]
Reviewed all the above changes. The content looks great. I will merge the pull request and then later create a new branch to update the screenshots to reflect "IBM Watson Studio".

**Pull request merged**